### ISSUE TITLE
Implement ls-ing a directory on OneDrive

### ISSUE THREAD

**Issue by Oleksii-Kshenskyi** [Oleksii-Kshenskyi/dark_hand#11]
This is a more minor convenience feature that would be nice to have, but is not that important for the overall run to succeed.

Implement ls-ing a directory on OneDrive. By "ls-ing", I mean functionality similar to the UNIX `ls` command, that is, showing the contents of a directory on OneDrive. If possible, this command should also show the size of all the files in the directory, that would be ideal.

This command should also have a command line interface. Something like `dark_hand ls <directory on OneDrive>` would do.

Once #8 is done, this can be implemented.

Use HTTPoison and probably also HTTP streams implemented in #7 to achieve this.
